.github/ISSUE_TEMPLATE/setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Setup
+about: Prepare for development with some basic project setup.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`). If child tickets are created for any line-item,
+please update this ticket description to include references to those tickets.
```

Comment:
```suggestion
Please check off line-items as they are completed and leave notes if necessary.
If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for any
line-item, please update this ticket description to include references to those
tickets.
```

Reply:
I stole this text from the `launch.yml` template. Want me to make this change there too?

Reply:
Sure!

Reply:
Done.